setup.py

```diff
@@ -51,6 +51,7 @@
         'console_scripts': [
             'datacube-ows=datacube_ows.wsgi:main',
             'datacube-ows-update=datacube_ows.update_ranges:main'
+            'datacube-ows-update_2=datacube_ows.update_ranges_2:main'
```

Comment:
```suggestion
            'datacube-ows-update-2=datacube_ows.update_ranges_2:main'
```
Mixing - and _ makes it hared to remember

Reply:
Agreed.  Need to look at why the checks are failing too.